README.md

```diff
@@ -2,10 +2,10 @@
 
 ![Banner](.wordpress-org/banner-1544x500.png)
 
-The official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+[Page Experience](https://developers.google.com/search/docs/guides/page-experience) (PX) is a set of ranking signals—including [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) (CWV)—measuring the user experience of interacting with a web page. AMP is a powerful and cost-effective tool enabling many optimizations and best practices automatically on your site, and making it easier for you to achieve good page experience. The official AMP Plugin, supported by the AMP team, makes it easy to bring the power of AMP to your WordPress site.
```

Comment:
Unfortunately, [according](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#example-readme) to the WP readme.txt spec, this can be a maximum of 150 characters and it cannot contain links.

In reality, the restriction seems to be even tighter than that. Only the first 15 words are displayed when browsing plugins:

![Listing](https://user-images.githubusercontent.com/134745/117054314-d1ead280-acce-11eb-9357-76714c2a48d5.png)

So most of this needs to be moved down under `## Description`.

Reply:
It could be reduced to just this:

```suggestion
An easier path to great Page Experience for everyone. Powered by AMP.
```

This is 12 words.

Reply:
I applied this in 7aac83230.